src/Data/Bool/Properties.agda

```diff
@@ -761,9 +760,7 @@ T-irrelevant : U.Irrelevant T
 T-irrelevant {true}  _  _  = refl
 
 T? : U.Decidable T
-does  (T? b) = b
-proof (T? true ) = ofʸ _
-proof (T? false) = ofⁿ λ()
+T? = Dec.T?
```

Comment:
I'm inclining towards reverting this, in favour of
```agda
T? b = b because (T⁺ b)
```
in order to decouple this change (and the need to reinstate the import of `Relation.Nullary.Reflects`) from any future possible refactoring/dependency simplification of `Data.Bool.Properties`?